tests/PactContinuationSpec.hs

```diff
@@ -676,7 +679,7 @@ testNestedPactYield = do
       chain1ContDupe <- makeContCmdWith 1 "chain1ContDupe"
 
       chain1Results <-
-        runAll' [moduleCmd, chain1Cont,chain1ContDupe] spv testFlags
+        runAll' [moduleCmd, chain1Cont,chain1ContDupe] spv nestedDefPactFlags
```

Comment:
Since nothing else has changed in this file, how is this name change OK? Are both symbols in scope here?

Reply:
This test was using the wrong flag. I noticed it. It came from copy paste